docs/source/release/v6.4.0/mantidworkbench.rst

```diff
@@ -8,46 +8,40 @@ Mantid Workbench Changes
 New Features
 ------------
 
-- Added the Fit Script Generator interface to the 'General' category of the interface menu.
-- New general setting: `Provide auto-completion suggestions in the script editor` for making auto-completion optional in the script editor
-- An additional memory bar has been added to the memory widget to enable users to see how much memory Mantid is using.
+- Added the :ref:`Fit Script Generator <Fit_Script_Generator-ref>` interface to the 'General' category of the interface menu.
+- Added a new general setting: ``Provide auto-completion suggestions in the script editor`` for making auto-completion optional in the script editor.
```

Comment:
I'm not sure this should be monospaced, as it isn't referencing code?

Reply:
Whilst it is note code in the strictest of senses it is the new option listed on the setting tab for users to use. It makes it more obvious that this is what the user is looking for in settings. I've realised I missed the formatting for General so I'll add that in.

Reply:
True, I just think it looks slightly strange for such a long string. Maybe _"Provide auto-completion suggestions in the script editor"_ would look better. Though I'm not overly fussed either way